### PR TITLE
grammatical error fixed

### PR DESCRIPTION
**Pull request by sanijcreates**
Fixes #

# 👩🏻‍💻 What does this PR do? 
[Grammatical error](https://github.com/openmsupply/open-msupply/tree/develop/server#serving-front-end) fix on the readme file of the server directory. 

# 🧪 How has/should this change been tested? 
no other changes than a change in the readme file, so didn't test.

### CODE REVIEW

File: server/configuration/base.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-      http://localhost:8000,
-    ] # Used to set the allowed Origin in Cross Origin Request Security
-  base_dir: "app_data"
-database:
-  host: "localhost"
-  port: 5432
-  username: "postgres"
-  password: "password"
-  database_name: "omsupply-database"
+# server:
```
Comment (roxy-dao) [inline server/configuration/base.yaml:1]: Can you revert this back?

**Review by roxy-dao — CHANGES_REQUESTED** [review]
LGTM~
Just one comment (:

**Comment by sanijcreates** [comment]
Absolutely, doing it right away! Apologies for such a silly mistake:)

**Comment by sanijcreates** [comment]
@roxy-dao I have reverted the changes on the base.yaml file. Please let me know if there is anything else I can do. Thank you for the help!

**Review by roxy-dao — APPROVED** [review]
Thanks~

File: node_modules/.yarn-integrity [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "systemParams": "win32-x64-108",
```
Comment (roxy-dao) [inline node_modules/.yarn-integrity:2]: Sorry can this be removed as well?

**Review by roxy-dao — CHANGES_REQUESTED** [review]
Sorry, missed something

**Review by roxy-dao — APPROVED** [review]
Thanks~

**Pull request merged**